lib/rubocop/cop/rspec/mixin/metadata.rb

```diff
@@ -38,7 +38,6 @@ def on_block(node)
             on_metadata_arguments(metadata_arguments)
           end
         end
-        alias on_numblock on_block
```

Comment:
This module is included in 4 cops and none have tests for numblocks. So I simply removed this one.

I don't think handling these blocks makes sense in context anyways and it was only added here because of the internal affairs cop

Reply:
I think it would be helpful to have some guidelines on when to add numblock/itblock aliases, and when it does not. It's too easy to appease toe internal affairs cop and just add a useless alias.